PixelToolkit.py

```diff
@@ -38,6 +38,13 @@
     wordlist_gen.add_argument("--min", type=int, help="Minimum keyword length", default=1)
     wordlist_gen.add_argument("--max", type=int, help="Maximum keyword length", default=100)
 
+    hash = subparsers.add_parser("hash", help="Text hashing")
+    hash.add_argument("--text", "-t", required=True, help="File or text to be hashed. Files will be hashed line by line.")
+    hash.add_argument("--algorithm", '-a', default="SHA256")
+    # User can now create file name to save the file.
+    hash.add_argument("--buf", "-b", required=True, help="Size of chunks", default=65536)
```

Comment:
as I said, this is a implementation detail which doesn't need to be exposed to the user

Reply:
I'll leave it as optional. What if file would be really large? Wouldn't it be better for user to choose how big chunks should be?

Reply:
this. is. an. implementation. detail. if we were talking about the hash's encoding buffer size, then yeah it makes sense to change that. in this case this buffer is just in what chunks python will read a file. it has nothing to do with hashing